apps/judicial-system/web/src/routes/Shared/CaseTable/CaseTable.tsx

```diff
@@ -215,17 +231,19 @@ const CaseTable: FC = () => {
           <>
             <Box display="flex" alignItems="center">
               <SectionHeading title={table.title} />
-              {table.hasMyCasesFilter && (
-                <Box marginBottom={3} marginLeft={'auto'}>
-                  <Checkbox
-                    label="Mín mál"
-                    checked={showOnlyMyCases}
-                    onChange={(e) => {
-                      setShowOnlyMyCases(e.target.checked)
-                    }}
-                  />
-                </Box>
-              )}
+              {table.hasMyCasesFilter &&
+                caseTableData &&
```

Comment:
Nit: we can also use a helper here to simplify the check -> `isNonEmptyArray(caseTableData.rows)`